pkg/service/controller/servingcert/secret_creating_controller.go

```diff
@@ -345,3 +347,12 @@ func (sc *ServiceServingCertController) requiresCertGeneration(service *v1.Servi
 	}
 	return true
 }
+
+func ownerRef(service *v1.Service) metav1.OwnerReference {
```

Comment:
can we make this a general helper like `MakeOwnerRef(runtime.Object) metav1.OwnerReference` ?

Reply:
> can we make this a general helper like MakeOwnerRef(runtime.Object) metav1.OwnerReference ?

Not in any obvious way.  Determining the `GroupVersion` is pretty unreliable since types can be serialized multiple ways.